src/app/pages/arbol-cuentas-contables/arbol-cuentas-contables.component.scss

```diff
@@ -1,61 +1,23 @@
-button[nbTreeGridRowToggle] {
-  background: transparent;
-  border: none;
-  padding: 0;
+.aligned {
+  display: flex;
+  justify-content: center;
 }
 
-.table-row {
-  &:hover {
-    background: #e6e6e6;
-  }
+.floating {
+  margin-bottom: 20px;
+  position: sticky;
+  top: 50px;
 
 }
-:host ::ng-deep table tr td {
-  padding: -0.125rem 1.25rem !important;
-  font-size: 12px !important;
-}
-.aligned-code {
-  padding-left: 8px;
-}
-.search-label {
-  display: block;
-}
-.search-input {
-  margin-bottom: 1rem;
-}
 
-.nb-column-name {
+:host ::ng-deep .mat-tab-label-content {
```

Comment:
Si se puede [evitar `::ng-deep`](https://dev.to/jwp/angular-10-ng-deep-and-host-pseudo-classes-1kbg), mejor, mas no es imprescindible